addon/components/em-select.js

```diff
@@ -32,8 +32,21 @@ export default Component.extend(InputComponentMixin, {
   autofocus: null,
   size: 0,
 
-  didReceiveAttrs() {
-    this._super(...arguments);
+  didRender() {
+    run.schedule('sync', this, () => {
```

Comment:
It's because it fail with older version of ember. It say we are changing a property in a didReceiveAttrs.